spec/requests/api/v1/events/nearest_spec.rb

```diff
@@ -0,0 +1,64 @@
+describe 'Events Requests' do
+  describe 'GET /v1/events/nearests?lat=&lon=&radius=' do
+    it 'returns events closest to lat and lon' do
+      near_event = create(:event,
+                          lat: 37.760322,
+                          lon: -122.429667)
+      farther_event = create(:event,
+                             lat: 37.760321,
+                             lon: -122.429667)
+      create(:event, lat: 37.687737, lon: -122.470608)
```

Comment:
That seems fine to me since most of your use case will be someone in a particular metro area, not someone globetrotting.